<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>კლიენტი</string>
    </property>
    <addaction name="add_client_action"/>
    <addaction name="edit_client"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="add_client_action">
   <property name="text">
    <string>კლიენტის დამატება</string>
   </property>
  </action>
  <action name="edit_client">
   <property name="text">
    <string>მონაცემების რედაქტირება</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
